docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -140,6 +153,9 @@ values. Typical usage does not require you to configure them.
     * - ``delta.target-max-file-size``
       - Target maximum size of written files; the actual size may be larger.
       - ``1GB``
+    * - ``delta.experimental.ignore-checkpoint-write-failures``
```

Comment:
@alexjo2144 @losipiuk i think we should remove the property.
But i also think we should ignore checkpoint writing failures: the table change has been committed (new tx file  created), so it's not quite fine to fail the query.

Reply:
I think yes..

Reply:
done

Reply:
Side note: I think such "behind the scenes" errors should be tracked via metrics. Then operation-wise one could plug in some monitoring service to those, and notice that something is going bad with the cluster, even though queries are not failing. 
Or maybe logging as WARN and allowing for logs monitoring is enough. WDYT?

Reply:
Good point. We don't have metrics infrastructure, do we, but monitoring over logging is indeed good idea.